### DISCUSSION TITLE
Add file tree to work with folder.

### DISCUSSION THREAD

**Discussion by alaireselene** [cpeditor/cpeditor#971] [Ideas]
Hi,
I just want to add a **file tree UI** to manage my file eazier like other text editors.
We can add/remove node like ``NERDtree`` vim plugin.
I hope we can see this feature soon.

**Comment by ouuan** [comment]
I'm wondering why you need a file tree in competitive programming :thinking: And I'm not familiar with NERDTree, what is "add/remove node"? Is it creating/deleting files and directories?

**Reply by alaireselene** [reply]
Yes, that's what I mean.
I want to open multiple file easier. Just open folder and switch between files. Moreover, our problem's names are not named like A, B, C, and I'm lazy to rename them.  Sorry for my bad English!

**Reply by ouuan** [reply]
Regardless of the problem names, why are you switching between files instead of solving them one by one?

**Reply by alaireselene** [reply]
Sound fine for me, thanks sir, but a quick rewind is also a good choice, isn't it?
